src/metaSurface.cxx

```diff
@@ -390,7 +390,14 @@ M_Read()
 
       for(d=0; d<4; d++)
       {
+      if (d+2*m_NDims < pntDim)
```

Comment:
The underlying problem is that `pntDim` is only 9, and not 10 as there are time number of this datum 3+3+4. This hack is simply to prevent the accessing `v` out of bounds.